docs/deployment/helm/README.md

```diff
@@ -0,0 +1,172 @@
+---
+Title: Deploying with Helm
+---
+
+# Deploying with Helm 
+Activiti Enterprise can be deployed using [Helm](https://helm.sh) charts. 
+
+## Prerequisites 
+A Helm deployment assumes that you have the following:
+
+* [Quay.io](https://quay.io) credentials to access Activiti Enterprise images. 
+* A valid license for Activiti Enterprise.
+* A Kubernetes cluster.
+* A Docker registry for the [deployment service](../architecture/platform.md#deployment-service).
+* Helm installed in the cluster. 
+* An ingress bound to an external DNS address.  
+
+## Deployment steps
+Activiti Enterprise deployment can be split into four areas: 
+
+1. Preparing the cluster:
+	* [Set up Quay.io credentials](#set-up-quayio-credentials)
+	* [Apply a license file to the cluster](#apply-a-license-to-the-cluster)
+2. [Set variables for the deployment](#set-the-variables-for-the-deployment)
+3. Configure any optional customizations:
+	* [Include Alfresco Content Services (ACS)](#optional-include-alfresco-content-services-acs)
+	* [Use external databases](#optional-use-external-databases)
+	* [Use a custom realm file](#optional-use-a-custom-realm-file)
+4. [Deploy to the cluster](#deploy-to-the-cluster) 
+
+### Set up Quay.io credentials 
```

Comment:
we should use 

```bash
kubectl create secret \
  docker-registry quay-registry-secret \
    --docker-server=quay.io \
    --docker-username="${DOCKER_REGISTRY_USER}" \
    --docker-password="${DOCKER_REGISTRY_PASSWORD}" \
    --docker-email="none"
```

Reply:
Replaced with updated quay instructions.